release-schema.json

```
@@ -120,8 +158,52 @@
             "boolean",
             "null"
           ]
+        },
+        "subcontracts": {
+          "title": "Subcontracts",
+          "description": "Information about the subcontracts that will be established.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Subcontract"
+          },
+          "uniqueItems": true,
+          "minItems": 1
+        }
+      },
+      "minProperties": 1
+    },
+    "Subcontract": {
+      "title": "Subcontract",
+      "description": "A relationship that the tenderers will establish with a third party to perform part of the contract on their behalf.",
```

Since we're defining Subcontract for use in multiple contexts, we should use the present tense, and the terms that are appropriate in that tense.

```suggestion
      "description": "A relationship that suppliers establish with a third party to perform part of the contract on their behalf.",
```